ibis/pandas/aggcontext.py

```diff
@@ -297,7 +297,11 @@ def compute_window_spec(dtype, obj):
 
 @compute_window_spec.register(type(None))
 def compute_window_spec_none(_, obj):
-    return obj
+    # Window spec in ibis is an inclusive window bound. A bound of 0 indicates
+    # the current row.
+    # Window spec in Pandas indicates window size. Therefore, we must add 1
+    # to the ibis window bound to get the expected behavior.
+    return obj + 1
```

Comment:
This only works with row window right?

Reply:
Yep, this is only used for row-based windows. There is another method on line 310 that handles windows defined by a time interval.

Added a comment here to clarify.

Reply:
@hjoo thanks for adding more details about this method.
could you provide the docstring using numpy format? https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Thanks @hjoo for the change!

I ran here pydocstyle for you change and it raised these 3 errors:

```
D400: First line should end with a period (not ':')
D401: First line should be in imperative mood; try rephrasing (found 'Helper')
```

it means that the first line should wne with a period (`.`) and should start in a verb in imperative mood like:
-  `Do ...`
- `Get ...`
- `Return ...`
- `Create ...`